FEXCore/Source/Interface/Core/JIT/Arm64/ConversionOps.cpp

```diff
@@ -449,5 +450,68 @@ DEF_OP(Vector_FToI) {
   }
 }
 
+DEF_OP(Vector_F64ToI32) {
+  const auto Op = IROp->C<IR::IROp_Vector_F64ToI32>();
+  const auto OpSize = IROp->Size;
+  const auto Round = Op->Round;
+
+  const auto Is256Bit = OpSize == Core::CPUState::XMM_AVX_REG_SIZE;
+
+  const auto Dst = GetVReg(Node);
+  const auto Vector = GetVReg(Op->Vector.ID());
+  if (HostSupportsSVE128 || HostSupportsSVE256) {
+    const auto Mask = Is256Bit ? PRED_TMP_32B.Merging() : PRED_TMP_16B.Merging();
+    // First step is to round the f64 values to integrals (frint*)
+    // Then convert to integers using fcvtzs.
+    switch (Round) {
+    case IR::Round_Nearest.Val:
+      frintn(ARMEmitter::SubRegSize::i64Bit, Dst.Z(), Mask, Vector.Z());
+      fcvtzs(Dst.Z(), ARMEmitter::SubRegSize::i32Bit, Mask, Dst.Z(), ARMEmitter::SubRegSize::i64Bit);
```

Comment:
Pop the fcvtzs out of the switch for deduplication across all the cases

Reply:
Popped